ecs-agent/api/resource/ebs_discovery_linux.go

```diff
@@ -78,13 +83,55 @@ func (api *EBSDiscoveryClient) ConfirmEBSVolumeIsAttached(deviceName, volumeID s
 //		   }
 //		]
 //	 }
-func parseLsblkOutput(output *LsblkOutput, deviceName string, volumeId string) (string, error) {
+//
+// If hasXenSupport is true then, in addition to matching a device by its serial, this function
+// matches a device by its name disregarding "sd" or "xvd" prefix (whichever is present).
```

Comment:
very tiny nit -- we don't disregard the sd/xvd.  We expect at least one to be present.  This reads a bit like we don't care if either exists.  I do see that the code does this explicit check:
```
if hasXenPrefix(block.Name) && hasXenPrefix(deviceName) {
```

Reply:
Yep. Updated in [034e758](https://github.com/aws/amazon-ecs-agent/pull/3971/commits/034e758b7e4f32b75cafe821fa28969e5a22a073).